crates/typst/src/layout/grid/cells.rs

```diff
@@ -411,8 +411,9 @@ impl<'a> CellGrid<'a> {
         let mut footer: Option<(usize, Span, Footer)> = None;
         let mut repeat_footer = false;
 
-        // Resolve the breakability of a cell, based on whether or not it spans
-        // an auto row.
+        // Resolves the breakability of a cell.
+        // Cells that span at least one auto-sized row or gutter
+        // are considered breakable.
```

Comment:
Please wrap at 80 columns. Also, please remove the linebreak between the first and second sentences.